Game/fr/Game.po

```diff
@@ -662,7 +663,7 @@ msgstr "Mode fenêtre"
 #: /Game/Resources/Localisation/ST_Menus.ST_Menus
 msgctxt "Menus,GameTag"
 msgid "ACT II"
-msgstr ""
+msgstr "ACTE II"
```

Comment:
Since the English here is understandable to a French speaker, to preserve the branding I would like to keep this the same